config/owasp/suppressions.xml

```diff
@@ -1,10 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.3.xsd">
-  <suppress until="2021-11-01">
-    <notes><![CDATA[
-   file name: spring-security-crypto-5.4.5.jar
-   ]]></notes>
-    <packageUrl regex="true">^pkg:maven/org\.springframework\.security/spring\-security\-crypto@.*$</packageUrl>
-    <cve>CVE-2018-1258</cve>
+  <suppress until="2022-05-13">
+    <cve>CVE-2022-23181</cve>
```

Comment:
please keep the `packageUrl`

Reply:
and is there way to avoid this vulnerability

Reply:
Updated dependencyManagement though I am a bit concerned about enforcing versions of such dependencies as tomcat.